### PR TITLE
Feature/search result page

### PR DESCRIPTION
**Pull request by rargotte**

### CODE REVIEW

File: src/containers/search-result-page/assets/js/script.js [hunk 1]
```diff
@@ -1,22 +1,68 @@
...
+        } else
+            myString = myString + "%2C" + element;
+    }
+    return (myString);
+}
 
+function buildQueryURL() {
+    queryURL = "https://api.themoviedb.org/3/discover/movie?api_key=" + APIKey;
+
+    language = "&language=" + language;
```
Comment (glazovg) [inline src/containers/search-result-page/assets/js/script.js]: we should be using `&with_original_language` instead `&language`

File: src/containers/search-result-page/assets/js/script.js [hunk 2]
```diff
@@ -42,23 +88,35 @@ function getMoviesFromAPI() {
         .then(function (data) {
 
             readMoviesArray(data.results);
+            console.log(queryURL)
```
Comment (glazovg) [inline src/containers/search-result-page/assets/js/script.js]: Is it necessary to leave `console.log`?

**Review by glazovg — CHANGES_REQUESTED** [review]

File: src/containers/search-result-page/assets/js/script.js (hunk 1 above)
Comment (rargotte) [reply src/containers/search-result-page/assets/js/script.js]: got it
Let me fix

File: src/containers/search-result-page/assets/js/script.js (hunk 2 above)
Comment (rargotte) [reply src/containers/search-result-page/assets/js/script.js]: absolutely not
all console.log should be deleted for final version.
Let me fix.

File: src/containers/search-result-page/assets/js/script.js (hunk 1 above)
Comment (rargotte) [reply src/containers/search-result-page/assets/js/script.js]: Done

File: src/containers/search-result-page/assets/js/script.js (hunk 2 above)
Comment (rargotte) [reply src/containers/search-result-page/assets/js/script.js]: I left console.log to check the URL.
Final version will be deleted.

**Review by glazovg — APPROVED** [review]

**Pull request merged**